google/cloud/pubsub_v1/subscriber/client.py

```diff
@@ -199,6 +200,15 @@ def callback(message):
                 *scheduler* to use when executing the callback. This controls
                 how callbacks are executed concurrently. This object must not be shared
                 across multiple SubscriberClients.
+            await_callbacks_on_shutdown (bool):
+                If ``True``, the returned future's ``close()`` method will block until
```

Comment:
```suggestion
                If ``True``, the returned future's ``cancel()`` method will block until
```

Reply:
The future's `cancel()` method has been specifically changed to _not_ block, it's the `result()` method that can now optionally block after canceling the future - will fix the docstring.